Setting-Up-Twitch.md

```diff
@@ -0,0 +1,25 @@
+## What is Twitch?
+
+Twitch is a live streaming video platform that was founded in June 2011. More than 1.7 million people broadcast on Twitch each month for 100 million unique visitors each month. Click [here](https://www.twitch.tv/p/about) to find out more about Twitch and it's statistics.
+
+## Setting up your Twitch Account
+
+Creating an account with Twitch enables you to interact with chat, get alerts when someone you know streams and maybe even stream yourself.
+
+1. To create a new account for Twitch go to this [page](https://passport.twitch.tv/users/new).
+
+2. You should now choose an appropriate username and password.
+![https://gyazo.com/5afeda27971f27fb448fa7a6b293e7b9](https://i.gyazo.com/5afeda27971f27fb448fa7a6b293e7b9.gif)
```

Comment:
- Add another blank line and add the image so that image appears just below the texts.
- Use a descriptive "alt" text (if required).
You may write it as 
```md
![](https://i.gyazo.com/5afeda27971f27fb448fa7a6b293e7b9.gif)
or
![username and password login](https://i.gyazo.com/5afeda27971f27fb448fa7a6b293e7b9.gif)
```